app/views/home/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<section class="home">
```

Comment:
- Kindly update your splash screen to follow the design guidelines and follow the guidelines:
   - [ ] The nav bar doesn't exist in the splash screen.
   - [ ] The App name has more spacing from the top and in the middle of the page.
   - [ ] The login and signup buttons are at the bottom of the page with a different button color and need more spacing from left and right.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/Christelle-12/Budget-manager/assets/24703171/05e3ba45-766e-455e-9180-0c4beda92697) | ![image](https://user-images.githubusercontent.com/24703171/219884305-f1437aeb-2f46-4261-81df-9ff3aaa904dd.png) |